index.php

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Movie</title>
+</head>
+<body>
+    <div class="homebar">
+        <h1 class="homepage">Welcome To My Movie Database</h1>
+</div>
+<div class="navbar">
+        <a class="home" href="http://localhost:1234/Collection-App/">Home</a>
+        <a class="Genre" href="#">Genre</a>
+        <a class="Add-title" href="#">Add Movie</a>
+    
+</div>
+
+
+
+<?php
+
+
+
+$db = new PDO('mysql:host=db; dbname=movies', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$inputtedCaregory =
+
+$query = $db->prepare('SELECT `movies`. `title`, `genre`.`name`, `movies`.`watched`, `movies`.`image`, `movies`.`about`
+                        FROM `movies`
+                        INNER JOIN `genre`
+                        ON `movies`.`genre_id` = `genre`.`id` ');
+
+
+$query->execute();
+
+$result = $query->fetchAll();
```

Comment:
Now you're at the point where you need to make a model class to contain this query.

You should make a MovieModel with a method called something like getAllMovies.

Take a look at the example we worked on Friday.

You should also think about using an Entity class to represent the Movies - again that example from Friday should help!